pkg/controller/scylladbdatacenter/resource.go

```diff
@@ -653,15 +663,30 @@ exec /mnt/shared/scylla-operator sidecar \
 --service-name=$(SERVICE_NAME) \
 --cpu-count=$(CPU_COUNT) \
 --loglevel=2 \
-` + func() string {
-										if len(sdc.Spec.ScyllaDB.ExternalSeeds) > 0 {
-											return fmt.Sprintf("--external-seeds=%s", strings.Join(sdc.Spec.ScyllaDB.ExternalSeeds, ","))
-										}
-										return ""
-									}(),
+` +
+										func() string {
+											var optionalArgs []string
+
+											if len(sdc.Spec.ScyllaDB.ExternalSeeds) > 0 {
+												optionalArgs = append(optionalArgs, fmt.Sprintf("--external-seeds=%s", strings.Join(sdc.Spec.ScyllaDB.ExternalSeeds, ",")))
+											}
+
+											return strings.Join(optionalArgs, ` \`)
+										}() +
+										func() string {
+											if len(positionalArgs) > 0 {
+												return ` -- "$@"`
+											}
+											return ""
+										}(),
 									),
 								}
 
+								if len(positionalArgs) > 0 {
+									cmd = append(cmd, "--")
+									cmd = append(cmd, positionalArgs...)
+								}
```

Comment:
nit: this should also be fine to set unconditionally

Reply:
I don't agree with previous comment, and I would leave this as is for consistency.